crates/edr_provider/tests/rip7212.rs

```diff
@@ -0,0 +1,49 @@
+#![cfg(feature = "test-utils")]
+
+use edr_eth::Bytes;
+use edr_evm::bytes;
+use edr_provider::{
+    test_utils::create_test_config, time::CurrentTime, MethodInvocation, NoopLogger, Provider,
+    ProviderRequest,
+};
+use edr_rpc_eth::CallRequest;
+use revm_precompile::secp256r1;
+use tokio::runtime;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn rip_7212() -> anyhow::Result<()> {
```

Comment:
Can we add another test that checks that the response is `0x` for the same calldata when the flag is false?

Reply:
Addressed in https://github.com/NomicFoundation/edr/pull/552/commits/25120887c4dfd21d494fce6b121d8c14310c8ef7